src/launch.c

```diff
@@ -67,17 +77,34 @@ main(int argc, char *argv[])
       setenv(ENV_LONG_TEST, "1", 1);
       cur_arg++;
     }
-    else if (strcmp(cur_arg[0], "--print-at-trace") == 0) {
-      setenv(ENV_PRINT_AT_TRACE, cur_arg[1], 1);
+    else if (strcmp(cur_arg[0], "--print-at-traceId") == 0 ||
+             strcmp(cur_arg[0], "-p") == 0) {
+      setenv(ENV_PRINT_AT_TRACE_ID, cur_arg[1], 1);
+      char *endptr;
+      if (strtol(cur_arg[1], &endptr, 10) != 0 || endptr[0] != '\0') {
+        fprintf(stderr, "%s: illegal value\n", "--print-at-traceId");
+        exit(1);
+      }
       cur_arg += 1;
     }
+    else if (cur_arg[0][1] == 'p' && isdigit(cur_arg[0][2])) {
+      setenv(ENV_PRINT_AT_TRACE_ID, cur_arg[0] + 2, 1);
+      cur_arg++;
+    }
+    else if (strcmp(cur_arg[0], "--quiet") == 0 ||
+             strcmp(cur_arg[0], "-q") == 0) {
+      setenv(ENV_QUIET, "1", 1);
+      cur_arg++;
+    }
     else if (strcmp(cur_arg[0], "--help") == 0 ||
              strcmp(cur_arg[0], "-h") == 0) {
-      fprintf(stderr,
-              "Usage: mcmini [--max-depth-per-thread|-m <num>] "
-              "[--debug-at-trace|-d <num>]\n"
-              "[--first-deadlock|-first] [--print-at-trace]\n"
-              "[--verbose|-v] [-v -v] [--help|-h] target_executable\n");
+      fprintf(stderr, "Usage: mcmini [--max-depth-per-thread|-m <num>]\n"
+                      "              [--print-at-traceId|-p <num>]\n"
+                      "              [--debug-at-traceId|-d <num>]\n"
+                      "              [--first-deadlock|-first]\n"
+                      "              [--verbose|-v] [-v -v]\n"
+                      "              [--help|-h]\n"
```

Comment:
`target_executable` should be on the same line at [--help|-h]`.

Reply:
@maxwellpirtle , If you feel strongly about this, tell me, and I'll change it to:
>                       "              [--help|-h] target_executable\n"

It seemed to me that since every other option was on a line by itself, conceptually, "target_executable" should be on a line by itself, and not mixed in with `[--help|-h]`.

But as I say, I'll make your change if you feel strongly about this.